scripts/deploy.py

```diff
@@ -150,6 +152,17 @@ def preprocess_release():
     with open(os.path.join('assets', 'constants.js'), 'w+') as new_assets_file:
         new_assets_file.write(content)
 
+    # Removes the version field from app.yaml.
+    print 'Removing the version field from app.yaml ...'
+    with open('app.yaml', 'r') as f:
+        content = f.read()
+        assert content.count('version: default') == 1
+    os.remove('app.yaml')
+    content = content.replace('version: default', '')
+    with open('app.yaml', 'w') as f:
```

Comment:
Oops I missed to follow up on this. What I was trying to say earlier was you might not need the os.remove(...) line, and this would directly overwrite it. Any specific reason we need the remove() line? Other than that, feel free to merge this!

Reply:
Oh, I see -- I was actually just putting back the actual code that we used to have prior to February or so. Both ways probably work, but I think I'll leave it as-is since the current version has served us well in so many releases. Thanks for spotting it!